db/error-handlers/error.js

```diff
@@ -1,13 +1,19 @@
 exports.handle400s = (err, req, res, next) => {
-  if (err.code === "22P02") {
+  console.log(err);
+  if (
+    err.code === "22P02" ||
+    err.code === "23502" ||
+    err.constraint === "comments_author_fkey" ||
```

Comment:
These constraint parts are not needed. The codes are specific to each issue so are enough to find the error we are looking for